lib/getLatestRelease.js

```diff
@@ -13,58 +14,79 @@ if (process.env.GH_TOKEN) {
 
 const getOS = str => {
   if (includes(str, 'darwin') || includes(str, '.dmg')) {
```

Comment:
Could we also add a `includes(str, 'mac') || ` here? (see https://github.com/resin-io/etcher/releases )

EDIT: And then we can also re-use this function down on line 98 :smiley: